Dataloader Sample Project - Bounty (#4096)
graphql-dotnet/graphql-dotnet: docs2/site/docs/guides/dataloader.md

GraphQL.NET includes an implementation of Facebook's [DataLoader](https://github.com/facebook/dataloader) within the
[`GraphQL.DataLoader`](https://www.nuget.org/packages/GraphQL.DataLoader) NuGet package.

> Sample projects for the Default Dataloaders described at the Top of this Page and the [DI based DataLoaders](#di-based-data-loaders) can be found in the [GraphQL.Net source repository](https://github.com/graphql-dotnet/graphql-dotnet).
> 

Reviewer: ```suggestion
Sample projects demonstrating both default data loaders (as described below on this page) and [DI-based data loaders](#di-based-data-loaders) are available in the [GraphQL.NET source repository](https://github.com/graphql-dotnet/graphql-dotnet/samples). These projects cover examples of data loaders accessed via `IDataLoaderContextAccessor` as well as those implemented using dependency injection (DI).
```

## Setup

> 1. Register the Dataloader Services

Reviewer: ```suggestion
1. Register the DataLoader services

```

## Setup

1. Register the Dataloader Services
``` csharp

> 

Reviewer: ```suggestion
```

services.AddGraphQL(b => b
    .AddDataLoader()
    // other configurations
);
```

> 2Hook up your GraphQL schema to your IoC container.

Reviewer: ```suggestion
2. Hook up your GraphQL schema to your IoC container.
```

## Setup

> 1. Register the Dataloader Services

Reviewer: Renumber the rest of the numbering in the sequence since 2 was deleted

``` csharp

> services.AddDataLoader();

Reviewer: ```suggestion
services.AddGraphQL(b => b
    .AddDataLoader()
    // other configurations
);
```
